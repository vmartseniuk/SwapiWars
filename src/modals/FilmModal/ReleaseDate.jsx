/* eslint-disable react/prop-types */
/* eslint-disable react/jsx-one-expression-per-line */
import React, { memo } from "react";

import { makeStyles } from "@material-ui/core";

const useStyles = makeStyles((theme) => ({
  realiseDate: {
    ...theme.typography.tiny,
    color: "#DBFF00",
    [theme.breakpoints.down("sm")]: {
      padding: "5px 0 20px",
    },
  },
}));
const ReleaseDate = (props) => {
  const classes = useStyles();
  const { children } = props;

  return <div className={classes.realiseDate}>Release date:{children}</div>;
};

export default memo(ReleaseDate);
