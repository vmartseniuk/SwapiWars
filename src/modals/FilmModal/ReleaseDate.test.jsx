import { create } from "react-test-renderer";
import ReleaseDate from "./ReleaseDate";

describe("FilmModal/ReleaseDate", () => {
  it("renders correctly", () => {
    const rendered = create(<ReleaseDate>2020:11:05</ReleaseDate>);
    expect(rendered.toJSON()).toMatchSnapshot();
  });
});
