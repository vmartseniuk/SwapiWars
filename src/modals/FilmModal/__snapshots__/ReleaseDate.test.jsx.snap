// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmModal/ReleaseDate renders correctly 1`] = `
<div
  className="makeStyles-realiseDate-1"
>
  Release date:
  2020:11:05
</div>
`;
